### DISCUSSION TITLE
Does wire-schema support kotlin/native?

### DISCUSSION THREAD

**Discussion by Kshitij09-sc** [square/wire#2631] [Q&A]
I'm planning to write a CLI using kotlin/native to validate proto files. Can we wire-shema for this?

**Comment by oldergod** [comment]
Actually, there's only targets for the JVM and JavaScript. But it should be easy to add targets for native.

**Reply by Kshitij09-sc** [reply]
@oldergod Any blocker for this? Are we using something only available on JVM or Js?

**Reply by oldergod** [reply]
There is just no need for us to have one right now. There's only a few classes to override. I can give it a try next week

**Reply by oldergod** [reply]
https://github.com/square/wire/pull/2635
